spaceflights/{{ cookiecutter.repo_name }}/conf/base/parameters.yml

```diff
@@ -1,11 +1 @@
-test_size: 0.2
-random_state: 3
-features:
-  - engines
-  - passenger_capacity
-  - crew
-  - d_check_complete
-  - moon_clearance_complete
-  - iata_approved
-  - company_rating
-  - review_scores_rating
+
```

Comment:
```suggestion
```
Only cause the file below shows as "empty file", rather than with one blank line.